app/femr/common/ItemModelMapper.java

```diff
@@ -176,8 +176,11 @@ public PatientItem createPatientItem(int id,
                                                 Float weight,
                                                 String pathToPatientPhoto,
                                                 Integer photoId,
-                                                String ageClassification) {
-
+                                                String ageClassification,
+                                                Integer smoker,
+                                                Integer diabetic,
+                                                Integer alcohol) {
+        // Osman above
         if (StringUtils.isNullOrWhiteSpace(firstName) ||
```

Comment:
Make sure to actually set the `smoker`, `diabetic`, and `alcohol` values in the `PatientItem` being created here.